recipe/meta.yaml

```diff
@@ -1,35 +1,40 @@
 {% set name = "pymultinest" %}
 {% set version = "2.11" %}
-{% set sha256 = "b853d3757875419827a1fd361c054f33db24b7360c1ef5db6dc90d2ee51e07ec" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
can you leave this and only change sha256 above?

Reply:
Ideally, no, b/c this makes it easier to update with the conda-forge automated tools like grayskull. Which I'm using here. This puts this recipe in compliance with the modern format.

If you feel strongly about this feel free to revert but I'll be unlikely to contribute more b/c it will become harder for me. I use as much automation as I can in order to be able to help with all the feedstocks in conda-forge.